src/js/test/rpc/server.test.ts

```diff
@@ -517,7 +517,7 @@ describe("Server", function () {
     it("shouldn't enable coprocessor if the coprocessor has invalid topics", () => {
       const server = new ProcessBatchServer();
       const coprocessor = createHandle().coprocessor;
-      coprocessor.inputTopics = [["topic.", PolicyInjection.Stored]];
+      coprocessor.inputTopics = [["topic%", PolicyInjection.Stored]];
```

Comment:
This test validates the topic name, but  `.` is valid, change for `%` it is a invalid topic

Reply:
Can you add a comment above the changed line saying that?

Reply:
Done